src/Language/PureScript/Errors.hs

```diff
@@ -1195,6 +1197,15 @@ renderBox = unlines
   dropWhileEnd p = reverse . dropWhile p . reverse
   whiteSpace = all isSpace
 
+evalTypelevelFunctions :: Type -> Type
```

Comment:
You definitely shouldn't need to kind check anything. I think this is fine, but we should add a check to make sure we're showing a `TypeLevelString`.